### PR TITLE
remove some unused top level variables

### PR DESCRIPTION
**Pull request by devoncarew**
- remove some unused top level variables from analytics/constants.dart

This PR removes a number of unused top-level fields from the `lib/src/analytics/constants.dart` library. I believe these were used in a previous iteration of the analytics impl.? I changed the imports of this library to use an import prefix - constants.dart has a number of symbols w/ short names, and it's useful to know where they come from.

### CODE REVIEW

File: packages/devtools_app/lib/src/app.dart [hunk 1]
```diff
@@ -504,7 +505,8 @@ class DevToolsAboutDialog extends StatelessWidget {
     final colorScheme = Theme.of(context).colorScheme;
     return InkWell(
       onTap: () async {
-        ga.select(devToolsMain, feedbackLink);
+        ga.select(
+            analytics_constants.devToolsMain, analytics_constants.feedbackLink);
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/app.dart:509]: nit: trailing comma

File: packages/devtools_app/lib/src/app.dart [hunk 2]
```diff
@@ -448,7 +448,8 @@ class ReportFeedbackButton extends StatelessWidget {
       tooltip: 'Report feedback',
       child: InkWell(
         onTap: () async {
-          ga.select(devToolsMain, feedbackButton);
+          ga.select(analytics_constants.devToolsMain,
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/app.dart:451]: nit: trailing comma

File: packages/devtools_app/lib/src/analytics/constants.dart [hunk 3]
```diff
@@ -3,38 +3,27 @@
...
 
 // Type of events (event_category):
-const String applicationEvent = 'application'; // visible/hidden
+
 const String screenViewEvent = 'screen'; // Active screen (tab selected).
 const String selectEvent = 'select'; // User selected something.
 
-const String errorError = 'onerror'; // Browser onError detected in DevTools
-const String exceptionEvent = 'exception'; // Any Dart exception in DevTools
+//const String exceptionEvent = 'exception'; // Any Dart exception in DevTools
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/analytics/constants.dart:10]: can you file an issue to track exceptions in analytics and link to the issue in a todo here?

**Review by kenzieschmoll — APPROVED** [review]

File: packages/devtools_app/lib/src/analytics/constants.dart (hunk 3 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/analytics/constants.dart:11]: Yup, will do.

File: packages/devtools_app/lib/src/app.dart (hunk 2 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/app.dart]: done

File: packages/devtools_app/lib/src/app.dart (hunk 1 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/app.dart]: done

**Review by devoncarew — COMMENTED** [review]
updated -

**Pull request merged**